xds/src/main/java/io/grpc/xds/CdsLoadBalancer2.java

```diff
@@ -183,7 +178,7 @@ private void handleClusterDiscovered() {
         helper.updateBalancingState(TRANSIENT_FAILURE, new ErrorPicker(unavailable));
         return;
       }
-      String endpointPickingPolicy = root.result.lbPolicy;
+      String endpointPickingPolicy = root.result.lbPolicy();
```

Comment:
In case of ring_hash, endpointPickingPolicy should be appended with `_experimental`.
 `new PolicySelection(endpointPickingLbProvider, null)` should not pass a non-null config.
It's fine if you do it when implementing the balancer.